index.js

```diff
@@ -124,7 +124,13 @@ module.exports = {
       get(groupId, (err, { secret, root }) => {
         if (err) return cb(err)
 
-        const recps = [groupId, ...feedIds]
+        const feedIdUris = feedIds
+          .map((feedId) =>
+            isFeedSSBURI(feedId) ? feedId : fromFeedSigil(feedId)
```

Comment:
what happens if I pass in the feedId `'dog'` ... it's not an ssb-uri, so we try to convert it. But it fails... how does it fail? is it null, does it throw.

Either way, I want that to be caught and for encryption to fail? I think that might be covered by like ssb-box2 though. 
ok.... lets leave it to the encryption to catch invalid inputs... hmm yes

Reply:
haven't checked now but i think it outputs null which in turn will get caught by schema checking